test/conversions.ts

```diff
@@ -92,7 +92,7 @@ describe('Conversion of a Gregorian calendar date to JDC', () => {
         const gregorianCalendarDate: CalendarDate = new CalendarDate(1582, 10, 4, undefined, 0);
         const jdc: number = JDNConvertibleConversionModule.gregorianToJDC(gregorianCalendarDate);
 
-        checkJDC(2299149.5, jdc);
+        checkJDC(2299159.5, jdc);
     });
 
     it('convert the Gregorian Calendar date 15-10-1582 to JDC', () => {
```

Comment:
the comment `15-10-1582` does not fit `new CalendarDate(-1001, 12, 22, undefined, 0)`. I guess this was always wrong, I'll fix it.